keps/sig-scheduling/1923-prefer-nominated-node/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-scheduling
 participating-sigs:
   - sig-scheduling
-status: provisional
+status: implementable
 creation-date: 2020-09-29
 reviewers:
```

Comment:
You are missing prr-approvers

Reply:
see my comment above, I don't think this is needed, what we need is a file under `prod-readiness/<sig>/<KEP number>.yaml`.

Reply:
please see: https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/1194-prod-readiness#phase-2---implementation

Reply:
The template still has it. But @wojtek-t  would know better.

Reply:
added, thanks!